apps/examples/ble_rmc/ble_rmc_main.c

```diff
@@ -258,10 +344,37 @@ int ble_rmc_main(int argc, char *argv[])
 		RMC_LOG(RMC_CLIENT_TAG, "deinit done[%d]\n", ret);
 	}
 
+	if (strncmp(argv[1], "reconn", 7) == 0) {
+		RMC_LOG(RMC_CLIENT_TAG, "== Try Auto Connect ==\n");
+
+		ble_bonded_device_list dev_list[BLE_MAX_BONDED_DEVICE] = { 0, };
+		uint16_t dev_count = 0;
+		ble_addr *addr;
+
+		ret = ble_manager_get_bonded_device(dev_list, &dev_count);
```

Comment:
`ret` is not used here. Please add checking `ret`
```
if (ret == BLE_MANAGER_SUCCESS) {
   RMC_LOG(RMC_CLIENT_TAG, "Bonded Dev Num : %d\n", dev_count);
  if (dev_count > 0) {
     addr = &(dev_list[0].bd_addr);
      ...
```

Reply:
Ok, I will fix it